### PR TITLE
Pass along kwargs in reset

### PR DESCRIPTION
**Pull request by strin**
TL;DR extend reset to be reset(**kwargs) so we can have reset the environment to a given configuration.

See PR: openai/universe#122

Also a related PR at `openai/gym`: https://github.com/openai/gym/pull/533

@joschu

### CODE REVIEW

**Comment by strin** [comment]
One problem with this implementation: `env.configure` will also send `env.reset` (see https://github.com/openai/universe/blob/master/universe/rewarder/rewarder_session.py#L198) This shortcuts the first call to reset, and users have no way to add kwargs to the first reset.

Proposed solution: change `env.configure` so that it only connects to env, but does not reset. It seems `gym` is not using `configure` at all, and we'll move away from them eventually.

@gdb @joschu

File: universe/vectorized/vectorize_filter.py [hunk 1]
```diff
@@ -25,8 +25,8 @@ def _configure(self, **kwargs):
         super(VectorizeFilter, self)._configure(**kwargs)
         self.filter_n = [self.filter_factory(*self._args, **self._kwargs) for _ in range(self.n)]
 
-    def _reset(self):
-        observation_n = self.env.reset()
+    def _reset(self, **kwargs):
```
Comment (gdb) [inline universe/vectorized/vectorize_filter.py]: You need to make this into a "def reset" right?

Comment (gdb) [inline universe/vectorized/vectorize_filter.py]: Oh I see.

Comment (strin) [inline universe/vectorized/vectorize_filter.py]: I made `reset` for all wrappers in `universe`.

**Comment by strin** [comment]
Good to merge?

**Comment by gdb** [comment]
It'd probably be cleaner to migrate to `def reset` everywhere rather than `def _reset`, but this is ok if you don't want to go down that road.

**Comment by strin** [comment]
Agree. We can conform to `reset` for new wrappers, and clean up this someday.

**Pull request closed without merging**